File: java/118A.java
// 118 A. String Task

import java.util.Scanner;

class Main {
    static boolean isVowel(char c) {
        c = Character.toLowerCase(c);
        return (c == 'a' || c == 'o' || c == 'y' || c == 'e' || c == 'u' || c == 'i');
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        StringBuilder result = new StringBuilder();

        for(char c : s.toCharArray()) {
            if(!isVowel(c)) {
                result.append('.');
                result.append(Character.toLowerCase(c));
            }
        }
        System.out.println(result.toString());
    }
}

File: cpp/118A.cpp
// 218 A. String Task

#include <bits/stdc++.h>
using namespace std;

bool isVowel(char c) {
    c = tolower(c);
    return (c == 'a' || c == 'o' || c == 'y' || c == 'e' || c == 'u' || c == 'i');
}

int main() {
    string s;
    cin >> s;
    string result = "";

    for (char c : s) {
        if (!isVowel(c)) {
            result += '.';
            result += tolower(c);
        }
    }

    cout << result << endl;
    return 0;
}